### ISSUE TITLE
how to get Renderpeople test dataset?

### ISSUE THREAD

**Issue by Holmes-Alan** [YuliangXiu/ECON#65]
Great work! How do you collect the Renderpeople test set as PIFu? Do you buy 3D models from Renderpeople for testing?

**Comment by YuliangXiu** [comment]
Yes, we bought RenderPeople 3D models, you could check the list in [testing.md](https://github.com/YuliangXiu/ECON/blob/master/docs/testing.md) and contact with RenderPeople Inc.
